src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -127,11 +127,22 @@ public async Task<uint256> SendTransactionAsync(string hex)
         /// RPC method that gets a new address for receiving payments.
         /// Uses the first wallet and account.
         /// </summary>
+        /// <param name="account">Parameter is deprecated.</param>
+        /// <param name="addressType">Address type, currently only 'legacy' is supported.</param>
         /// <returns>The new address.</returns>
         [ActionName("getnewaddress")]
         [ActionDescription("Returns a new wallet address for receiving payments.")]
-        public NewAddressModel GetNewAddress()
+        public NewAddressModel GetNewAddress(string account, string addressType)
```

Comment:
I wonder if we should we create an overload for this method?

Reply:
If it's called via RPC no need as the parameters are optional so don't need to be included. If we want to call directly, then maybe. The wrapper in RPCClient actually doesn't pass these parameters at all.